packages/backend/src/index.ts

```diff
@@ -41,14 +41,16 @@ const DEFAULT_PORT = 7000;
 const PORT = parseInt(process.env.PORT ?? '', 10) || DEFAULT_PORT;
 
 function createEnv(plugin: string): PluginEnvironment {
-  return {
-    logger: getRootLogger().child({ type: 'plugin', plugin }),
-    database: knex({
-      client: 'sqlite3',
-      connection: ':memory:',
-      useNullAsDefault: true,
-    }),
-  };
+  const logger = getRootLogger().child({ type: 'plugin', plugin });
+  const database = knex({
+    client: 'sqlite3',
+    connection: ':memory:',
+    useNullAsDefault: true,
+  });
+  database.client.pool.on('createSuccess', (_eventId: any, resource: any) => {
```

Comment:
Maybe it's worth breaking the database creation out into a seperate function? That could also be used in tests.

Reply:
Maybe soon - this is not really production code anyway, it's here temporarily for the in memory database until we actually start talking to postgres.

Reply:
`backend-test-utils` inc